lib/generators/decidim/templates/app.json.erb

```diff
@@ -0,0 +1,11 @@
+{
+  "name": "<%= app_name %>",
+  "description": "TODO: Add a short description about <%= app_name %>",
+  "keywords": [
+    "small",
+    "sharp",
+    "tool"
+  ],
+  "addons": [],
```

Comment:
Even though postgres is automatically detected, I would explicitly add it with a plan. You can use `decidim.barcelona`'s `app.json` as an example:

https://github.com/AjuntamentdeBarcelona/decidim.barcelona/blob/master/app.json